File: Practice(Beginner)/C++/Mahasena.cpp
// Source: https://www.codechef.com/problems/AMR15A

#include <bits/stdc++.h>
using namespace std;

int main() {

    int n, temp, odd = 0, even = 0;
    cin >> n;
    for (int i = 0; i < n; ++i) {
        cin >> temp;
        temp%2 == 0 ? even++ : odd++;
    }
    even > odd ? cout << "READY FOR BATTLE" : cout << "NOT READY";
    return 0;
}

File: Practice(Beginner)/Python3/Mahasena.py
# Source: https://www.codechef.com/problems/AMR15A

n = int(input())
l = list(map(int,input().split()))
evn,odd = 0,0
for i in l:
    if(i%2==0):
        evn += 1
    else:
        odd += 1
if(evn > odd):
    print("READY FOR BATTLE")
else:
    print("NOT READY")
